File: BackEnd/postman/Telefono.py
import requests
def postear(data, url):
    return requests.post('http://127.0.0.1/syme/BackEnd/services/'+url+'.php', data=data).content

url='Telefono'
data={
  'accion':'agregarTelefonoPersona',
  'idPersona': None,
  'idPais': None,
  'telefono': None,
  'idTipoTelefono': None,
}
print postear(data,url)
print

url='Telefono'
data={
  'accion':'listarTipoTelefono',
}
print postear(data,url)
print

url='Telefono'
data={
  'accion':'buscarPorPersona',
  'pNombre': None,
  'noIdentidad': None,
  'pApellido': None,
}
print postear(data,url)
print

url='Telefono'
data={
  'accion':'listarPorCentroMedico',
}
print postear(data,url)
print

url='Telefono'
data={
  'accion':'listarPorPersona',
}
print postear(data,url)
print

url='Telefono'
data={
  'accion':'actualizarTipoTelefono',
}
print postear(data,url)
print

url='Telefono'
data={
  'accion':'agregarTipoTelefono',
  'tipoTelefono': None,
}
print postear(data,url)
print

url='Telefono'
data={
  'accion':'agregarTelefonoCentro',
  'idPais': None,
  'idCentroMedico': None,
  'telefono': None,
  'idTipoTelefono': None,
}
print postear(data,url)
print

url='Telefono'
data={
  'accion':'eliminarDePersona',
  'idPersona': None,
  'idTelefono': None,
}
print postear(data,url)
print

url='Telefono'
data={
  'accion':'eliminarDeCentro',
  'idTelefono': None,
  'idCentroMedico': None,
}
print postear(data,url)
print

url='Telefono'
data={
  'accion':'buscarPorCentro',
  'idCentroMedico': None,
}
print postear(data,url)
print

File: BackEnd/ajax/Telefono.js
$.ajax({
  url:SITIO_URL+'/services/Telefono.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'agregarTelefonoPersona',
    'idPersona': null,
    'telefono': null,
    'idTipoTelefono': null,
    'idPais': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Telefono.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarTipoTelefono',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Telefono.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'buscarPorPersona',
    'noIdentidad': null,
    'pNombre': null,
    'pApellido': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Telefono.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorCentroMedico',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Telefono.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorPersona',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Telefono.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'actualizarTipoTelefono',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Telefono.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'agregarTipoTelefono',
    'tipoTelefono': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Telefono.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'agregarTelefonoCentro',
    'idCentroMedico': null,
    'telefono': null,
    'idTipoTelefono': null,
    'idPais': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Telefono.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'eliminarDePersona',
    'idPersona': null,
    'idTelefono': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Telefono.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'eliminarDeCentro',
    'idCentroMedico': null,
    'idTelefono': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:SITIO_URL+'/services/Telefono.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'buscarPorCentro',
    'nombreCentro': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


